tests/e2e/vsphere_volume_expansion.go

```diff
@@ -846,19 +857,54 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
 
-		ginkgo.By("Create POD using the above PVC")
-		pod, _ := createPODandVerifyVolumeMount(f, client, namespace, pvclaim, volHandle)
+		if supervisorCluster {
+			ginkgo.By("Trigger offline volume expansion on PVC on shared VMFS datastore")
```

Comment:
are you clubbing offline and online resize in the same test?

Reply:
yes